code/game/jobs/job/logistics/engi/ordnance_tech.dm

```diff
@@ -33,4 +33,4 @@ AddTimelock(/datum/job/logistics/otech, list(
 /obj/effect/landmark/start/otech
 	name = JOB_ORDNANCE_TECH
 	icon_state = "ot_spawn"
-	job = /datum/job/logistics/otech
+	job = /datum/job/logistics/otech
```

Comment:
Not sure why this file was edited, but it's where the error is

Reply:
For some reason my 'master' version was excluding the otech spawn landmark, so to solve the problem I copied the script lines from the branch in github and glued them on my files, which actually solved the problem, but looks like it have been...too much glued

Reply:
I just had forgotten to jump a line in the end, heh